graphistry/pygraphistry.py

```diff
@@ -270,7 +270,8 @@ def _handle_auth_url(auth_url, sso_timeout, sso_opt_into_type):
         if in_ipython() or in_databricks() or sso_opt_into_type == 'display':  # If run in notebook, just display the HTML
             # from IPython.core.display import HTML
             from IPython.display import display, HTML
-            display(HTML(f'<a href="{auth_url}" target="_blank">Login SSO</a>'))
+            display(HTML(f'<a href="{auth_url}" target="_blank">old: Login SSO</a>'))
+            display(Markdown(f'[new: Login SSO]({auth_url})"))
```

Comment:
missing import? + ci fails

does  this work in both jupyter + databricks?

Reply:
also, `old/:new:` seems kind of funny for the text label, i think that's just for testing